solidity/contracts/KeepBonding.sol

```diff
@@ -371,11 +375,15 @@ contract KeepBonding {
 
         unbondedValue[operator] = unbondedValue[operator].sub(amount);
 
-        (bool success, ) = tokenStaking.beneficiaryOf(operator).call.value(
-            amount
-        )("");
+        address beneficiary = tokenStaking.beneficiaryOf(operator);
+        require(
```

Comment:
Can we add a comment for future-us that this situation happens only when someone deposited ETH in `KeepBonding` before defining the beneficiary?